package.json

```diff
@@ -45,13 +45,16 @@
   "dependencies": {
     "big-integer": "1.6.28",
     "big.js": "5.1.2",
+    "body-parser": "1.18.3",
     "caporal": "0.10.0",
     "cli-table": "0.3.1",
     "colors": "1.2.4",
     "compare-versions": "3.1.0",
+    "express": "4.16.3",
     "generate-safe-id": "1.0.2",
     "grpc": "1.11.3",
     "grpc-caller": "0.5.0",
+    "grpc-dynamic-gateway": "sparkswap/grpc-dynamic-gateway#55d5b29f043e24ce31c9f52ae1c95fa9533e5440",
```

Comment:
we can remove this dep since we've imported the file instead. We will need to make sure we have the correct dependencies